environment.js

```diff
@@ -2,25 +2,37 @@ var path = require('path');
 var fs = require('fs');
 var mkdirp = require('mkdirp');
 var glob = require('glob');
+var q = require('q');
 
-export.ensureDirectory = function(directoryName) {
+exports.ensureDirectory = function(directoryName) {
   if (!fs.existsSync(directoryName)) {
     mkdirp.sync(directoryName);
   }
 };
 
-export.loadSpecifications = function(mocha, directory) {
-   function addSpecs(error, files) {
+
+exports.loadSpecifications= function(directory, mocha) {
+  var deferred = q.defer();
```

Comment:
I'm not sure due to the error handling situation, I felt this had the right separation of responsibilities between the call and the library code.